src/shared/components/videoCard/VideoCard.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import "./videoCard.css";
+import {
+  BsThreeDotsVertical,
+  MdWatchLater,
+  BsCollectionPlayFill,
+} from "../../../assets/icons";
+import { useExplore } from "../../../context/explore/ExploreContext";
+import ReactPlayer from "react-player";
+import { DropDown } from "../dropDown/Dropdown";
+
+export const VideoCard = (product) => {
+  const { _id, title, description, creator, views, releaseDate } = product;
+
+  const options = [
+    {
+      item: (
+        <div>
+          <BsCollectionPlayFill className="mr-1" /> Add to playlists
+        </div>
+      ),
+      value: "playlist",
+    },
+    {
+      item: (
+        <div>
+          <MdWatchLater className="mr-1" /> Add to watch later
+        </div>
+      ),
+      value: "watchLater",
+    },
+  ];
+
+  const { selectedDropdownId, dispatch } = useExplore();
+  return (
+    <div className="video-card-container flex-col">
+      <div className="video-wrapper">
+        <ReactPlayer
+          className="react-player"
+          url={`https://www.youtube.com/watch?v=${_id}`}
+          width="100%"
+          height="100%"
+          light={false}
```

Comment:
fix: You can remove this light attribute

Reply:
I will remove it .